src/app/components/ui/Fab/__tests__/Fab.stories.js

```diff
@@ -2,18 +2,108 @@ import React from 'react';
 import { storiesOf } from '@storybook/react';
 
 import Fab from '../';
-import AddIcon from '@material-ui/icons/Add';//USE LOCAL
+import { styled } from 'styletron-react';
+import Icon from 'app/components/ui/Icon';//USE LOCAL
+
+const PaddedContainer = styled('div', ({black}) => {
+  return {
+    display: 'flex',
+    fontSize: '20px',
+    flexWrap: 'wrap',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    background: (black && 'black') || 'inherit',
+    color: (black && 'white') || 'inherit'
+  };
+});
+
+const IconHolder = styled('div', {
+  margin: '20px',
+  width: '150px',
+  flexDirection: 'column',
+  display: 'flex',
+  justifyContent: 'center',
+  textAlign: 'center'
+});
+
+const icons = [
+  "",
+  "acciones",
+  "aire",
+  "alerta",
+  "arrow-drop-down",
+  "arrow-drop-up",
+  "arrow_left",
+  "arrow_less",
+  "arrow_more",
+  "arrow_right",
+  "atras",
+  "borrar",
+  "caldera",
+  "calendario",
+  "camara",
+  "cerrar",
+  "cif",
+  "clip",
+  "cuadre-de-caja",
+  "defectos",
+  "direccion",
+  "dto",
+  "editr",
+  "estado-aviso",
+  "factura",
+  "filtro",
+  "firma",
+  "historial",
+  "info",
+  "intervencion",
+  "listado",
+  "lupa",
+  "mail",
+  "mantenimiento",
+  "menu",
+  "movil",
+  "observaciones",
+  "papelera",
+  "perfil",
+  "phone",
+  "piezas",
+  "salir",
+  "servicios",
+  "sort",
+  "toggle-off",
+  "toggle-on",
+  "usuario",
+  "ver"
+  ];
 
 storiesOf('FAB', module)
   .add('Primary', () => (
     <Fab color="primary" aria-label="Add">
-      <AddIcon />
+      <Icon/>
     </Fab>
   ))
   .add('Secondary', () => (
     <Fab color="secondary" aria-label="Add">
-      <AddIcon />
+      <Icon/>
     </Fab>
+  ))
+  .add('Icons', () => (
+    <PaddedContainer >
+      {
+        icons.map( (icon, ind) => (
+        <IconHolder key={ind}>
+          <Fab color={'primary'} style={{margin: 'auto'}}>
+            <Icon
```

Comment:
When there are few attributes one line improve readability. 
```suggestion
            <Icon fab icon={icon} />
```

Reply:
Done.